content/guides/autodiscovery.md

```diff
@@ -278,8 +278,52 @@ spec:
         - containerPort: 80
 ~~~
 
+
+### Template Source: Docker Label Annotations
+
+Since version 5.17 of the Datadog Agent, you can store check templates in Docker labels. With Autodiscovery enabled, the Agent detects if it's running on Docker and automatically searches all labels for check templates; you don't need to configure a template source (i.e. via `SD_CONFIG_BACKEND`) as you do with key-value stores.
```

Comment:
"Since version...." sounds awkward to me.  I know this was copied from previous sections, but can we adjust?

Reply:
Perhaps "as of version x..."